rdkPlugins/Minidump/source/Minidump.cpp

```diff
@@ -146,18 +147,25 @@ std::string Minidump::getDestinationFile()
     std::stringstream timeString;
     timeString << std::put_time(std::localtime(&currentTime), "%FT%T");
     std::string destFile;
+    std::string fileName;
 
     std::string dir(mContainerConfig->rdk_plugins->minidump->data->destination_path);
 
     std::map<std::string, std::string> annotations = mUtils->getAnnotations();
 
     auto it = annotations.find(FIREBOLT_STATE);
     if (it != annotations.end()) {
-        destFile = dir + "/" + mUtils->getContainerId() + MINIDUMP_FN_SEPERATOR + it->second.c_str() + MINIDUMP_FN_SEPERATOR + timeString.str() + ".dmp";
+        fileName = mUtils->getContainerId() + "_" + it->second.c_str() + "_" + timeString.str();
+        if (fileName.length() > MINIDUMP_FILENAME_LENGTH)
+            fileName.resize(MINIDUMP_FILENAME_LENGTH);
+        destFile = dir + "/" + fileName + ".dmp";
```

Comment:
the 70 char limit includes the directory and the .dmp extension.
so the limit for filename is actually (70-dir.length()-4)
if dir is /opt/secure/minidumps, then limit becomes 70 - 22 - 4 = 44 chars.